From 517a11d8c1eb8d490dc204672fffad6ca26f3ceb Mon Sep 17 00:00:00 2001
From: Saul Wold <sgw@linux.intel.com>
Date: Wed, 25 Jan 2017 10:35:42 -0800
Subject: [PATCH] ver_linux: Use /usr/bin/awk instead of /bin/awk

Most Linux distributions contain awk in /usr/bin by default, not
in /bin.

Signed-off-by: Saul Wold <sgw@linux.intel.com>
Signed-off-by: Bruce Ashfield <bruce.ashfield@windriver.com>
---
 scripts/ver_linux | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)
 mode change 100755 => 100644 scripts/ver_linux

diff --git a/scripts/ver_linux b/scripts/ver_linux
old mode 100755
new mode 100644
index b51de8a7e2a3..f8c33adf2fea
--- a/scripts/ver_linux
+++ b/scripts/ver_linux
@@ -1,4 +1,4 @@
-#!/usr/bin/awk -f
+#!/usr/usr/bin/awk -f
 # Before running this script please ensure that your PATH is
 # typical as you use for compilation/installation. I use
 # /bin /sbin /usr/bin /usr/sbin /usr/local/bin, but it may
-- 
2.5.0

